modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheSharedManagerAdapter.java

```diff
@@ -174,13 +167,6 @@ protected String stopInfo() {
         return "Cache manager stopped.";
     }
 
-    /**
-     * @return Start info.
-     */
-    protected String kernalStartInfo() {
```

Comment:
Perhaps it's worth to add onKernelStart callback logging for symmetry with onKernelStop/start/stop

Reply:
It's about a refactoring, not a cleanup. 
As you may see, 20+ (scopes #2 + #3) files still need an initial dummy cleanup.
Let's do this before starting any refactoring.